src/signals/incident-management/containers/IncidentDetail/components/MapDetail/index.js

```diff
@@ -4,7 +4,7 @@ import Map from 'components/Map';
 
 import './style.scss';
 
-const MapDetail = ({ value, hideAttribution, hideZoomControls, zoom }) => {
+const MapDetail = ({ value, hideAttribution, hideZoomControls, useSmallMarker, zoom }) => {
```

Comment:
All props you don't use in the component. Try passing them with `...props` syntax.